.github/workflows/test-cli-with-database.yml

```diff
@@ -71,6 +71,14 @@ jobs:
           --output medusa-config.js
         working-directory: ../cli-test
 
+      - name: Install sqlite config
```

Comment:
question: I am not sure we really well support sqlite since the migrations are written in SQL most of the time. wdyt?